code/06-variables/Makefile

```diff
@@ -9,7 +9,7 @@ results.txt : $(ZIPF_SRC) *.dat
 dats : isles.dat abyss.dat last.dat
 
 %.dat : books/%.txt $(COUNT_SRC)
-	$(COUNT_EXE) $< $*.dat
+	$(COUNT_EXE) $< $@
```

Comment:
As above, this hasn't been suggested in the lesson at this point, so I think it should be left as it was.